src/vcSettingsUI.cpp

```diff
@@ -940,6 +940,23 @@ void vcSettingsUI_BasicMapSettings(vcState *pProgramState)
     vcSettings_ApplyMapChange(&pProgramState->settings);
     vcRender_ClearTiles(pProgramState->pRenderContext);
   }
+
+  if (ImGui::Button(vcString::Get("settingsMapECEFMode")))
+  {
+    static const int32_t s_ECEFModeID = 4978;
```

Comment:
This constant fits better as a value in the `vcProjectStandardZones` enum- `vcPSZ_WGS84ECEF`.

There are other places I've used `4978` as a magic number that will need to change eventually as well but that's outside the scope of this PR.